composeApp/src/commonMain/kotlin/com/arkivanov/minesweeper/game/GameContent.kt

```diff
@@ -126,6 +131,43 @@ internal fun GameContent(component: GameComponent, modifier: Modifier = Modifier
     }
 }
 
+@Composable
+fun RemainingMines(
```

Comment:
I think we can leverage the `padStart` function and do something as follows. This `Counter` function could be reused in both cases - remaining mine counter and (future) time counter.

```kotlin
@Composable
private fun Counter(value: Int, modifier: Modifier = Modifier) {
    Row(modifier = modifier, horizontalArrangement = Arrangement.Center) {
        value.toCounterString().forEach { char ->
            Image(
                modifier = Modifier.size(width = 13.dp, height = 23.dp),
                painter = LocalGameIcons.icons.digits.getValue(char),
                contentDescription = char.toString(),
            )
        }
    }
}

private fun Int.toCounterString(): String =
    if (this >= 0) {
        coerceAtMost(999).toString().padStart(length = COUNTER_LENGTH, padChar = '0')
    } else {
        "-" + coerceAtLeast(-99).absoluteValue.toString().padStart(length = COUNTER_LENGTH - 1, padChar = '0')
    }
```